### PR TITLE
lets deprecate old exposecontroller flags now that we recommend `jx u…

### PR DESCRIPTION
**Pull request by rawlingsj**
…pgrade ingress`

#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/config/helm_values.go [hunk 1]
```diff
@@ -97,6 +97,9 @@ func (c *HelmValuesConfig) AddExposeControllerValues(cmd *cobra.Command, ignoreD
 	cmd.Flags().StringVarP(&c.ExposeController.Config.TLSAcme, "tls-acme", "", "", "Used to enable automatic TLS for ingress")
 	cmd.Flags().BoolVarP(&keepJob, "keep-exposecontroller-job", "", false, "Prevents Helm deleting the exposecontroller Job and Pod after running.  Useful for debugging exposecontroller logs but you will need to manually delete the job if you update an environment")
 
+	cmd.Flags().MarkDeprecated("http", "please use `jx upgrade ingress` after install instead")
+	cmd.Flags().MarkDeprecated("tls-acme", "please use `jx upgrade ingress` after install instead")
+
```
Comment (ccojocar) [inline pkg/config/helm_values.go]: @jstrachan Should we also add `helm3` to the list and recommend `no-tiller`? It seems to be broken just saw this issue this morning https://github.com/jenkins-x/jx/issues/2951. We don't maintain the fork of helm3.

Comment (jstrachan) [reply pkg/config/helm_values.go]: good idea!

**Pull request merged**